web/src/classic/components/molecules/Visualizer/WidgetAlignSystem/MobileZone.tsx

```diff
@@ -40,16 +41,19 @@ export default function MobileZone({
   pluginBaseUrl,
   isEditable,
   isBuilt,
+  invisibleWidgetIDs,
   onWidgetAlignAreaSelect,
   children,
   ...props
 }: Props) {
   const filteredSections = useMemo(() => {
     return sections.filter(
       s =>
-        areas.filter(a => zone?.[s]?.[a]?.widgets?.length).length || (s === "center" && children),
+        areas.filter(a => zone?.[s]?.[a]?.widgets?.find(w => !invisibleWidgetIDs?.includes(w.id)))
```

Comment:
IMO: `some` is better, but `find` is also ok.
```suggestion
        areas.filter(a => zone?.[s]?.[a]?.widgets?.some(w => !invisibleWidgetIDs?.includes(w.id)))
```

Reply:
Updated. Thank you!